core/src/test/scala/org/apache/spark/ShuffleSuite.scala

```diff
@@ -419,6 +424,31 @@ abstract class ShuffleSuite extends SparkFunSuite with Matchers with LocalSparkC
 
     manager.unregisterShuffle(0)
   }
+
+  test(s"SPARK-34541: shuffle can be removed when spark.shuffle.useOldFetchProtocol=true") {
```

Comment:
nit: `SPARK-34541: shuffle files should be removed normally`

(`spark.shuffle.useOldFetchProtocol=true` is not valid for `ShuffleSuite`.)

Reply:
`ShuffleSuite`  is abstract class, ShuffleOldFetchProtocolSuite would cover the `spark.shuffle.useOldFetchProtocol=true` case.

Reply:
I mean, the test name is not correct for other extended shuffle suites. For other suites, `spark.shuffle.useOldFetchProtocol` should be `false`, right?